docs/EndToEndTests/Main.md

```diff
@@ -22,3 +28,33 @@ These tests deal with all issues related to the main screens of the app.
 ### Special Considerations
 
 - These tests need to run before and after the project naming tests.
+
+## GPS
+
+This tests the GPS pill that is on the main map screen and the main camera screen.
+
+### Test Objectives
+
+- On home **Map** screen, tests that GPS details button shows current device status
+- On home **Camera** screen, tests that GPS details button shows current device status
+- Tests that **GPS details** button from home **Map** screen navigates to **GPS details** screen
+- Tests that **GPS details** button from **Home** camera screen navigates to **GPS details** screen
+- **GPS details** screen shows detailed navigation data of device
+- Tests that the **Back** button returns to last home screen (**Map** or **Camera**)
+
+### Special Considerations
+
+- This test sets the location to x before the test
```

Comment:
is there a specific coordinate that "x" represents?

Reply:
Oops! Good catch. Sorry.